.github/workflows/libvmaf.yml

```diff
@@ -75,6 +75,15 @@ jobs:
       - name: Run tests
         run: |
           sudo ninja -vC libvmaf/build test
+      - name: Run meson asan
+        run: |
+          LDFLAGS=-fsanitize=address,undefined meson setup libvmaf libvmaf/build_asan --buildtype debug -Db_sanitize=address,undefined -Denable_float=true
+      - name: Run ninja asan
+        run: |
+          sudo ninja -vC libvmaf/build_asan install
+      - name: Run tests asan
```

Comment:
Are these tests a superset of "Run tests" above? as in, is there a way that `Run tests asan` could pass but `Run tests` could fail? I'm thinking that `Run tests` may be redundant

Reply:
The optimized binaries are available for download after the whole thing is done, that's why we do both. The latency/compute overhead should be small.